index.html

```diff
@@ -103,15 +103,21 @@ <h3>
         </h3>
         <pre class="idl">
           partial interface Navigator {
+            [SecureContext] boolean canShareFiles(sequence&lt;USVString&gt; file_types);
```

Comment:
We probably just want a regular `DOMString`. 

```suggestion
            [SecureContext] boolean canShareFiles(sequence&lt;DOMString&gt; file_types);
```